src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -24,6 +24,8 @@ namespace Microsoft.Build.BackEnd.Logging
     /// <remarks>This class is not thread safe.</remarks>
     internal class ParallelConsoleLogger : BaseConsoleLogger
     {
+        private bool _cancellationMessageRendered;
```

Comment:
What's the downside of rendering this multiple times? I am not sure whether it's worth keeping the bool around.

Reply:
I saw the message written twice when I hit Ctrl+C. I don't remember details and I'm not able to repro it now, so it was probably fixed by different change. I will remove it.